drivers/hmis_csv_twenty_twenty/app/models/hmis_csv_twenty_twenty/hmis_csv_cleanup/delete_empty_enrollments.rb

```diff
@@ -0,0 +1,48 @@
+###
+# Copyright 2016 - 2020 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/master/LICENSE.md
+###
+
+module HmisCsvTwentyTwenty::HmisCsvCleanup
+  class DeleteEmptyEnrollments < Base
+    def cleanup!
+      enrollment_batch = []
+
+      enrollments_with_no_service = enrollment_scope.where.not(id: enrollment_scope.joins(:services).select(:id))
+      enrollments_with_no_cls = enrollment_scope.where.not(id: enrollment_scope.joins(:current_living_situations).select(:id))
```

Comment:
I think these scopes should be different.  The first just for ES TrackingMethod=3, the second only for SO.